### PR TITLE
chore(docs): Update docs based on feedback

### PR DESCRIPTION
**Pull request by abdallahshaban557**
Update docs based on feedback

### CODE REVIEW

**Comment by abdallahshaban557** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #69** <a href="https://app.graphite.dev/github/pr/celest-dev/website/69?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/celest-dev/website/69?utm_source=stack-comment).

File: docs/get-started.md [hunk 1]
```diff
@@ -19,13 +19,26 @@ To use Celest in your Flutter app, you need the following prerequisites:
 That’s it! You are now ready to start building your backend - all in Dart!
 
 ## Starting a new Celest project
-After installing the Celest CLI, navigate to the root of your Flutter project in your console and run the following command.
+
+Start by first creating a new Flutter project. You can also just add Celest to an existing Flutter project. To start a new Flutter project, go to your console and run the following command.
```
Comment (dnys1) [inline docs/get-started.md:23]: Start by first creating a new Flutter project. To start a new Flutter project, go to your console and run the following command.

\> If you have an existing Flutter project, you can use that instead.

File: docs/get-started.md [hunk 2]
```diff
@@ -43,6 +56,66 @@ flutter_app/
     └── test/               # Tests for your backend
 ```
 
+To get started building your serverless cloud function, navigate to the `<flutter_app>/celest/functions/` folder and create a file named `<api_name>.dart`. You can create as many APIs as you want in this directory. Think of each file as a way to organize and group multiple Celest Functions of similar functionality into a single namespace.
```
Comment (dnys1) [inline docs/get-started.md:59]: single namespace -> namespace

File: docs/get-started.md [hunk 3]
```diff
@@ -43,6 +56,66 @@ flutter_app/
...
+That's all you need to define your API! Now, you can connect your Flutter app to your cloud functions by using the code-generated client as shown in the following example. Replace the contents of the `main.dart` file in your Flutter app to the following code-snippet.
+
+```dart
+import 'package:flutter/material.dart';
+// Import the code-generated Celest client
+// highlight-next-line
+import 'package:celest_backend/client.dart';
+
+void main() {
+  // initializes Celest in your Flutter app
```
Comment (dnys1) [inline docs/get-started.md:82]: initializes -> Initializes

File: docs/get-started.md [hunk 4]
```diff
@@ -43,6 +56,66 @@ flutter_app/
...
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Homepage'),
+        ),
+        body: Center(
+          child: FutureBuilder(
+            // Call your Celest Function using the code-generated client
```
Comment (dnys1) [inline docs/get-started.md:100]: Call your Celest Function using the client

**Review by dnys1 — APPROVED** [review]
Minor tweaks. LGTM!

**Pull request merged**